lib/puppet/provider/network_config/redhat.rb

```diff
@@ -181,7 +181,7 @@ def self.format_file(filename, providers)
     props    = {}
 
     # Map everything to a flat hash
-    props = (provider.options || {})
+    props = provider.options if provider.options and provider.options != :absent
```

Comment:
https://travis-ci.org/voxpupuli/puppet-network/jobs/115005624#L369
rubocop says:  `Use && instead of and.`